### DISCUSSION TITLE
Filtering

### DISCUSSION THREAD

**Discussion by clivejefferies** [OWASP/threat-dragon#858] [Ideas] [answered]
You can end up with a large number of threats for a system. It would be good to be able to filter them by status, priority and type.

**Comment by jgadsden** [comment]
Would this be implemented in the Threat Dragon report?

**Comment by clivejefferies** [comment, marked as answer]
Hi, I'm talking about the threat list (see image) 
![Screenshot 2024-02-06 at 14 39 22](https://github.com/OWASP/threat-dragon/assets/973791/97206f91-2ff6-4010-ad29-326c73306cdb)

**Comment by jgadsden** [comment]
What sort of solution would you like to see @clivejefferies ? Is this reordering according to priority, say, or only seeing high priority ones?

**Reply by clivejefferies** [reply]
Hi @jgadsden, yeh it would be seeing only those selected.
